qa/prscript.py

```diff
@@ -165,7 +165,7 @@ def OpenBuildbotSession():
     opener = urllib2.build_opener(urllib2.HTTPCookieProcessor(cookie))
     urllib2.install_opener(opener)
     request = urllib2.Request(BASE_URI + 'login', params)
-    page = urllib2.urlopen(request)
+    urllib2.urlopen(request)
```

Comment:
I suggest using `_ = urllib2.urlopen(request)` instead to make it clear that a return value (1) is available and (2) explicitly ignored.

Reply:
Good idea